src/pages/2020/components/Carousel/index.js

```diff
@@ -71,6 +70,8 @@ const Carousel = ({ children }) => {
 							className={
 								current === i
 									? [styles.slide, styles.currentSlide].join(' ')
+									: current === (i+1) && type === 'images' ? [styles.slide, styles.prevSlide].join(' ')
```

Comment:
if we had variants here it would be easier to re-use the styles — so instead of switching the classname, you're switching the variant property in `sx`

just something to think about, not necessarily change right now

Reply:
Yeah that would be better. Right now I'm not even using those class names for anything. just testing/making sure we can target the next/previous slides.